src/tslint.ts

```diff
@@ -14,106 +14,100 @@
+        const configuredRules = Lint.loadRules(configuration, enableDisableRuleMap, rulesDirectory);
+        for (let rule of configuredRules) {
+            if (rule.isEnabled()) {
+                const ruleFailures = rule.apply(sourceFile);
+                for (let ruleFailure of ruleFailures) {
+                    if (!this.containsRule(failures, ruleFailure)) {
+                        failures.push(ruleFailure);
                     }
                 }
             }
+        }
 
-            let formatter: Lint.IFormatter;
-            const formattersDirectory = this.getRelativePath(this.options.formattersDirectory);
-
-            const Formatter = Lint.findFormatter(this.options.formatter, formattersDirectory);
-            if (Formatter) {
-                formatter = new Formatter();
-            } else {
-                throw new Error("formatter '" + this.options.formatter + "' not found");
-            }
+        let formatter: Lint.IFormatter;
+        const formattersDirectory = this.getRelativePath(this.options.formattersDirectory);
 
-            const output = formatter.format(failures);
-            return {
-                failureCount: failures.length,
-                failures: failures,
-                format: this.options.formatter,
-                output: output
-            };
+        const Formatter = Lint.findFormatter(this.options.formatter, formattersDirectory);
+        if (Formatter) {
+            formatter = new Formatter();
+        } else {
+            throw new Error("formatter '" + this.options.formatter + "' not found");
         }
 
-        private getRelativePath(directory: string): string {
-            if (directory) {
-                return path.relative(moduleDirectory, directory);
-            }
+        const output = formatter.format(failures);
+        return {
+            failureCount: failures.length,
+            failures: failures,
+            format: this.options.formatter,
+            output: output
+        };
+    }
 
-            return undefined;
+    private getRelativePath(directory: string): string {
+        if (directory) {
+            return path.relative(moduleDirectory, directory);
         }
 
-        private containsRule(rules: RuleFailure[], rule: RuleFailure) {
-            return rules.some((r) => r.equals(rule));
-        }
+        return undefined;
+    }
+
+    private containsRule(rules: Lint.RuleFailure[], rule: Lint.RuleFailure) {
+        return rules.some((r) => r.equals(rule));
     }
 }
 
-// add the Lint and TypeScript modules to global for pluggable formatters/rules
-global.Lint = Lint;
-global.ts = ts;
+import {findConfiguration as config} from "./configuration";
 
-// export Lint.Linter as the API interface for this module
-module.exports = Lint.Linter;
-// also export Lint.Configuration.findConfiguration so implementers can consume
-module.exports.findConfiguration = Lint.Configuration.findConfiguration;
+namespace Linter {
```

Comment:
It's not n ambiguity - they're merged. This syntax is the only way to make an interface a static member of a class. If we want `import Linter = require('tslint');` to continue working as it does today, this merging is the best (only?) way to do it. `export default` isn't liable to work well in node with TS, since we fixup our exports with `__esModule`, but don't check for it on imports. `LintResult` and `ILinterOptions` are still reexported into the `Lint` namespace elsewhere.